### PR TITLE
Resolves ASC-14: Create OIDC auth plugin

### PR DESCRIPTION
**Pull request by gkostin1966**

### CODE REVIEW

**Comment by ssciolla** [comment]
Thanks! Can you merge your other PR and then rebase?

File: app/Dockerfile [hunk 1]
```diff
@@ -64,6 +57,20 @@ RUN apt-get update && \
...
+
+RUN cd /archivesspace && \
+    ./scripts/initialize-plugin.sh aspace-oauth
+
+# Install AppConfig file
+COPY config.rb /archivesspace/config/config.rb
+
+# Install Start Up Script
+COPY startup.sh /archivesspace/startup.sh
+RUN chmod u+x /archivesspace/startup.sh
```
Comment (ssciolla) [inline app/Dockerfile:72]: Could you elaborate a little on your thinking for moving these into the latter stage? Is it just that they don't leave any build garbage behind, so why not do it in the final stage?

**Review by ssciolla — APPROVED** [review]
Looks good! I'd like to see the commit history tidied up, but I'll leave it up to you how you want to merge it.

File: app/Dockerfile (hunk 1 above)
Comment (gkostin1966) [reply app/Dockerfile:72]: I moved them to speed up development iteration since all they do is add additional code.

**Pull request merged**